### DISCUSSION TITLE
Add rail-based public transport view

### DISCUSSION THREAD

**Discussion by kurb70** [organicmaps/organicmaps#9001] [Ideas]
Hi! Maybe I've missed something, but it would be nice if there was a view where the railway lines (and trams/subways) with their stops are highlighted and the rest fade into the background. This would make it clear which areas are easily accessible by public transport and where there is a dense network of rail-based public transport.

**Comment by MrSuddenJoy** [comment]
@kurb70 I agree. Would be nice to have rail-based PT view.

**Reply by biodranik** [reply]
Please follow/update #1577
